frontend/src/api/actions/init.js

```diff
@@ -80,6 +80,9 @@ export function initFromStage(stage, options = { startAfterStage: false }) {
     return async (dispatch, getState) => {
         const parsedGlobals = { mockAPI: null, activeSession: null };
 
+        // These actions don't need to finish in a specific order, so we can wait for them to finish at the end of this function to speed up startup.
+        const asyncActions = [fetchInstructors];
```

Comment:
Still no. This doesn't save any time.

You need to dispatch the action right away, but only await the action at the end.

Reply:
so line 84 should be: `const asyncActions = [dispatch(fetchInstructors())];`
and then at the bottom should be `await Promise.all(asyncActions);`?

Reply:
Yes